src/vs/workbench/contrib/debug/browser/debugANSIHandling.ts

```diff
@@ -126,44 +131,165 @@ export function handleANSIOutput(text: string, linkDetector: LinkDetector, theme
 	}
 
 	/**
-	 * Calculate and set basic ANSI formatting. Supports bold, italic, underline,
-	 * normal foreground and background colors, and bright foreground and
+	 * Swap foregfrouned and background colors.  Used for color inversion.  Caller should check
```

Comment:
Typo, should be "foreground"

Reply:
fixed this typo.  I should have caught something that blatant. lol